assembl/static2/css/components/switchButton.scss

```diff
@@ -182,3 +182,47 @@ $rsbc-theme-flat-color-disabled-border : darken($rsbc-theme-flat-color-disabled,
     }
   }
 }
+
+// switchButton style for the administration
```

Comment:
too bad we don't have the same style (or something similar) elsewhere...
you can add a story for this one too. Just copy the 3 `SwitchButton` stories that I wrote in `stories/common.jsx` add wrap the switch button in a `div.administration` should do the trick